weave/type_wrappers/Content/content.py

```diff
@@ -80,21 +45,29 @@ class Content(BaseModel, Generic[T]):
     content_type: ContentType
     input_type: str
 
+    encoding: str = Field(
+        "utf-8", description="Encoding to use when decoding bytes to string"
+    )
+
     metadata: Annotated[
         dict[str, Any] | None,
         Field(
             description="metadata metadata to associate with the content",
             examples=[{"number of cats": 1}],
         ),
     ] = None
-    encoding: str | None = "utf-8"
     path: str | None = None
     extension: str | None = None
 
     _last_saved_path: Annotated[
         str | None, Field(description="Last path the file was saved to")
     ] = None
 
+    # These fields are set by serialization layer when it picks up a pydantic class
+    # We define them here so they can be set without doing `extra=allow`
+    _ref: Annotated[Any, PrivateAttr] = PrivateAttr(None)
+    _art: Annotated[Any, PrivateAttr] = PrivateAttr(None)
```

Comment:
This should be whatever the Art type is or None

Reply:
Fixed. I was worried about circular imports but turns out its fine